po/fr.po

```diff
@@ -1,177 +1,178 @@
 #: src/views/CaesarCipherView.vala:54 src/views/Base64EncodingView.vala:49
 #: src/views/AtbashCipherView.vala:49 src/views/AsciiEncodingView.vala:49
 msgid "Decipher"
-msgstr ""
+msgstr "Déchiffrer"
 
 #: src/views/ROT13CipherView.vala:51 src/views/PolybiusSquareCipherView.vala:51
 #: src/views/HashFunctionsView.vala:63 src/views/CaesarCipherView.vala:64
 #: src/views/Base64EncodingView.vala:51 src/views/AtbashCipherView.vala:51
 #: src/views/AsciiEncodingView.vala:51
 msgid "Plain Text"
-msgstr ""
+msgstr "Texte brut"
 
 #: src/views/ROT13CipherView.vala:54 src/views/PolybiusSquareCipherView.vala:54
 #: src/views/CaesarCipherView.vala:68 src/views/Base64EncodingView.vala:54
 #: src/views/AtbashCipherView.vala:54 src/views/AsciiEncodingView.vala:54
 msgid "Cipher Text"
-msgstr ""
+msgstr "Texte chiffré"
 
 #: src/views/HashFunctionsView.vala:47
 msgid "Generate Hash"
-msgstr ""
+msgstr "Générer un Hash"
 
 #: src/views/CipherView.vala:76
 #: data/com.github.arshubham.cipher.appdata.xml.in:55
 msgid "Caesar Shift Cipher"
-msgstr ""
+msgstr "Chiffre de César"
 
 #: src/views/CipherView.vala:77
 #: data/com.github.arshubham.cipher.appdata.xml.in:56
 msgid "Atbash Cipher"
-msgstr ""
+msgstr "Atbash"
 
 #: src/views/CipherView.vala:78
 #: data/com.github.arshubham.cipher.appdata.xml.in:60
 msgid "Polybius Square Cipher"
-msgstr ""
+msgstr "Carré de Polybe"
 
 #: src/views/CipherView.vala:79
 #: data/com.github.arshubham.cipher.appdata.xml.in:57
 msgid "ROT13 Cipher"
-msgstr ""
+msgstr "ROT13"
 
 #: src/views/CipherView.vala:80
 #: data/com.github.arshubham.cipher.appdata.xml.in:59
 msgid "Base64 Encoding"
-msgstr ""
+msgstr "Encodage Base64"
 
 #: src/views/CipherView.vala:81
 msgid "Hash Functions"
-msgstr ""
+msgstr "Fonctions de Hash"
 
 #: src/views/CipherView.vala:82
 #: data/com.github.arshubham.cipher.appdata.xml.in:58
 msgid "ASCII Encoding"
-msgstr ""
+msgstr "Encodage ASCII"
 
 #: src/views/CaesarCipherView.vala:61
 msgid "Number of letters to shift to the right: "
-msgstr ""
+msgstr "Nombre de lettres à décaler vers la droite : "
 
 #: src/configs/Constants.vala:26 data/com.github.arshubham.cipher.desktop.in:6
 msgid "com.github.arshubham.cipher"
-msgstr ""
+msgstr "com.github.arshubham.cipher"
 
 #: src/configs/Constants.vala:28 data/com.github.arshubham.cipher.desktop.in:3
 #: data/com.github.arshubham.cipher.desktop.in:4
 #: data/com.github.arshubham.cipher.appdata.xml.in:7
 msgid "Cipher"
-msgstr ""
+msgstr "Cipher"
 
 #: src/configs/Constants.vala:31
 msgid "An simple app for encoding and decoding text"
-msgstr ""
+msgstr "Une simple application pour encoder et décoder du texte"
 
 #: src/widgets/HeaderBar.vala:88
 msgid "Light background"
-msgstr ""
+msgstr "Thème clair"
 
 #: src/widgets/HeaderBar.vala:91
 msgid "Dark background"
-msgstr ""
+msgstr "Thème sombre"
 
 #: src/widgets/HeaderBar.vala:131
 msgid "Click to learn more about "
-msgstr ""
+msgstr "Cliquez pour en savoir plus"
```

Comment:
Please add extra space here as cipher name will be inserted after this.
```suggestion
msgstr "Cliquez pour en savoir plus "
```

Reply:
Ah okay I am going to change that because in this case it should be:
msgstr "Cliquez pour en savoir plus sur "